integrations/oci/sdk/runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciAvailabilityDefault.java

```diff
@@ -46,7 +46,7 @@ static boolean runningOnOci(OciConfig ociConfig) {
             return false;
         }
 
-        return (Region.getRegionFromImds() != null);
+        return (Region.getRegionFromImds("http://" + ociConfig.imdsHostName() + "/opc/v2/") != null);
```

Comment:
What is the issue with calling the no arg method?  It seems like we should not need to pass the path.

If we really must take ownership of the path, then "/opc/v2/" should be configurable.  We should therefore follow-up to make that configurable - perhaps using `oci.yaml`.

Reply:
without the arg, it is returning null. It can be fixed by using Region.registerFromInstanceMetadataService() but this uses http://169.254.169.254/opc/v2/. Our code however allows IMDS host to be configurable via ociConfig.imdsHostName(), hence why we cannot use  Region.registerFromInstanceMetadataService(). The IMDS URL is in the format, http://<IMDS HOST>/opc/v2/ hence why I derived it and hence why "/opc/v2/" does not need to be configurable?